.github/workflows/main.yml

```diff
@@ -454,9 +454,8 @@ jobs:
           fgrep 'HTML documentation ... no' config.log
           fgrep 'man pages ... no' config.log
           fgrep 'ch-image(1) ... no' config.log
-          fgrep 'with Docker: yes' config.log
           fgrep 'with ch-image(1): no' config.log
-          fgrep 'basic tests, all stages: yes' config.log
+          command -v docker
```

Comment:
This is for the case where Docker is being used as the builder and `ch-image` is absent. In the current state of this PR, `ch-image` is the only builder that `configure` actually checks for. As a result, `basic tests, all stages` will report `no` unless `ch-image` is found (see [here](https://github.com/hpc/charliecloud/pull/1769/files#diff-49473dca262eeab3b4a43002adb08b4db31020d190caaad1594b47f1d5daa810L778)), which as I said previously isn't true for this CI test. I added `command -v docker` because I thought it was still a good idea to check for the builder we expect to have.